src/ui/src/js/projects.js

```diff
@@ -23,6 +23,8 @@ window.$ = window.jQuery = jQuery;
 
 import "jstree";
 
+require("datatables.net")(window, $);
```

Comment:
does this grab scripts from external sites?

Reply:
@MindTooth can probably answer this better. datatables is installed with npm and this indicate it is needed, similar to import, but the import syntax did not work for me. So I stayed with what worked, not using time to get it working.

Reply:
This syntax is called CommonJS and is to be regardes as obsolete.  Now we use what’s called ESM.  `import` is newer and should be used.

Regarding `.net`, the package has it in its name. 😅